test/passing/tests/js_source.ml.ref

```diff
@@ -169,7 +169,6 @@ end
 
 (* Structure items *)
 let%foo[@foo] x = 4
-
 and[@foo] y = x
```

Comment:
This is more consistent regarding the formatting of other items, the break should be doubled if there is a docstring and not an attribute like this.